src/main/webapp/js/components/dev/ExperimentsTable/ExperimentsTable.js

```diff
@@ -0,0 +1,524 @@
+	 * information (name, lastModified) and controls.
+	 */
+	var ExperimentRow = React.createClass({
+		//Requests element with control icons to become visible
+		mouseOver: function () {
+			this.refs.icons.show();
+		},
+		//Requests element with control icons to become invisible
+		mouseOut: function () {
+		   this.refs.icons.hide();
+		},
+		componentDidMount : function(){
+			var row = "#"+this.props.experiment.getId();
+			
+			// Handle edits to editable name field
+            $(row).find("td[contenteditable='true']").keydown(function(e)
+            {
+            	//save if user hits enter key
+                if (e.keyCode == 13)
+                {
+                    e.preventDefault();
+                    // get experiment ID for the edited field
+    				var val = $(this).html();
+    				var field = $(this).attr("name");
+
+    				var setterStr = "setName";
+
+    				if (field == "name")
+    				{
+    					var expID = $(this).parent().attr("id").replace('#', '');
+    					GEPPETTO.Console.executeCommand("Project.getExperimentById(" + expID + ")." + setterStr + "('" + val + "')");
+    				}
+
+                    // without this somehow the carriage return makes it
+                    // into the field
+                    window.getSelection().removeAllRanges();
+                }
+            });
+		},
+		render: function() {
+			var rowNumber = this.props.rowNumber;
+			var rowClasses = "experimentsTableColumn accordion-toggle row-"+this.props.experiment.getId();;
+			//add different background to every even row
+			if(rowNumber%2==1){
+				rowClasses += " nthTr";
+			}
+			//set editable name field if experiment in design ode
+			var editable = false;
+            if (this.props.experiment.getStatus() == GEPPETTO.Resources.ExperimentStatus.DESIGN){
+            	editable = true;
+            }
+			
+			return (
+					<tr rowType="main"  onClick={this.props.fnClick} onMouseOver={this.mouseOver} onMouseOut={this.mouseOut} 
+						className={rowClasses} id={this.props.experiment.getId()}>
+						<StatusElement experiment={this.props.experiment} key={this.props.experiment.name}/>
+						<td name="name" contentEditable={editable}>{this.props.experiment.getName()}</td>
+						<td>{this.props.experiment.getLastModified()}</td>
+						<td><IconsElement ref="icons" experiment={this.props.experiment} key={this.props.experiment.name}/></td>
+					</tr>
+			);
+		}
+	});
+	
+	/**
+	 * Creates a table row html element, <tr>, which can be collapsed.
+	 * Used to display an experiment's simulator configurations
+	 */
+	var ExperimentExpandableRow = React.createClass({
+		render: function() {
+			var id = "collapsable-"+this.props.experiment.getId();
+			var rowNumber = this.props.rowNumber;
+			var rowClasses = "nested-experiment-info"
+			//add different background to every even row
+			if(rowNumber%2==1){
+				rowClasses += " nthTr";
+			}
+			var style  = {
+				width:"215px"
```

Comment:
Curious about this extra style information -- does it not work to have this in the less file?